src/main/java/org/apache/commons/lang3/SystemUtils.java

```diff
@@ -1433,6 +1569,26 @@ static boolean isJavaVersionMatch(final String version, final String versionPref
     }
 
     /**
+     * Decides if the operating system and its major and minor version matches.
+     *
+     * <p>
+     * This method is package private instead of private to support unit test invocation.
+     * </p>
+     *
+     * @param osName the actual OS name
+     * @param osVersion the actual OS version
+     * @param osNamePrefix the prefix for the expected OS name
+     * @param osVersionPrefix the expected major and minor OS version, e.g. "10.10"
+     * @return true if matches, or false if not or can't determine
+     */
+    static boolean isOSMajorMinorVersionMatch(final String osName, final String osVersion, final String osNamePrefix, final String osVersionPrefix) {
+        if (osName == null || osVersion == null) {
+            return false;
+        }
+        return osName.startsWith(osNamePrefix) && osVersion.matches(osVersionPrefix + "(\\.\\d+)?");
```

Comment:
How is this different from ```isOSMatch(final String osName, final String osVersion, final String osNamePrefix, final String osVersionPrefix)``` ? Looks rather complicated to use are regex instead of simply ```String.startsWith(String)```